core/src/scene/styleParam.cpp

```diff
@@ -159,7 +160,7 @@ StyleParam::Value StyleParam::parseString(StyleParamKey key, const std::string&
     case StyleParamKey::style:
         return _value;
     case StyleParamKey::font_size: {
-        float fontSize;
+        float fontSize = 10.f;
```

Comment:
I remember that discussing with @tallytalwar to remove those here to keep the default style value parameters in the struct they belong to (just to look up in the same place to edit the defaults). Is there any other reason to add them here?